pennylane/plugins/default_qubit.py

```diff
@@ -431,6 +452,14 @@ def _get_operator_matrix(self, operation, par):
         return A(*par)
 
     def ev(self, A, wires):
+        r"""Expectation value of observable on specified wires.
+
+         Args:
+          A      (array[float]): the observable matrix as array
```

Comment:
Just for formatting consistency with all other docstrings in the library, best to remove the extra whitespace; i.e.,

```suggestion
          A (array[float]): the observable matrix as array
```

This avoids superfluous whitespace changes appearing in alternating PRs